src/libs/OptionsListUtils.js

```diff
@@ -599,6 +599,32 @@ function isCurrentUser(userDetails) {
     return _.some(_.keys(loginList), (login) => login.toLowerCase() === userDetailsLogin.toLowerCase());
 }
 
+/**
+ * Calculates count of all enabled options
+ *
+ * @param {Object[]} options - an initial strings array
+ * @param {Boolean} options[].enabled - a flag to enable/disable option in a list
+ * @param {String} options[].name - a name of an option
+ * @returns {Boolean}
+ */
+function getEnabledCategoriesCount(options) {
+    return _.chain(options)
```

Comment:
why not simple do?
```
    return options.filter((option) => option.enabled).length;
```

Reply:
We still has a lint rule for `.js` files.
<img width="668" alt="Screenshot 2023-09-19 at 11 30 37" src="https://github.com/Expensify/App/assets/57314631/a7c9a97f-2b87-4f8c-bcd8-8f6a258c2e1b">